### PR TITLE
Use all cores for Turbo builds

### PR DESCRIPTION
**Pull request by steveluscher**
I like using all 48 cores for these. How about you folks?

### CODE REVIEW

**Comment by steveluscher** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2799?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#2799** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2799?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#2798** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2798?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @steveluscher and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by steveluscher** [comment]
The other option is that we leave this the way it is (defaults to 10 cores) and I can just get used to writing `--concurrency=100%` when I run steps locally.

**Comment by buffalojoec** [comment]
> The other option is that we leave this the way it is (defaults to 10 cores) and I can just get used to writing `--concurrency=100%` when I run steps locally.

Do you even alias bro?

File: package.json [hunk 1]
```diff
@@ -28,7 +29,7 @@
         "@typescript-eslint/parser": "^6.21.0",
         "agadoo": "^3.0.0",
         "eslint": "^8.57.0",
-        "eslint-config-turbo": "^1.13.3",
+        "eslint-config-turbo": "^2.0.3",
```
Comment (buffalojoec) [inline package.json:32]: Are these package version bumps also necessary?

File: turbo.json [hunk 2]
```diff
@@ -1,6 +1,9 @@
 {
     "$schema": "https://turbo.build/schema.json",
-    "pipeline": {
+    "remoteCache": {
+        "signature": true
+    },
+    "tasks": {
```
Comment (buffalojoec) [inline turbo.json:6]: Assuming these config changes are for Turbo `2,0`?

**Review by buffalojoec — APPROVED** [review]
speed

File: package.json (hunk 1 above)
Comment (steveluscher) [reply package.json:32]: Uh. Wtf. These are already in. Maybe I didn't rebase this PR…

Comment (steveluscher) [reply package.json]: There. Rebased. That makes more sense.

**Pull request merged**